caching-service/src/main/java/org/infinispan/demo/online/Main.java

```diff
@@ -16,10 +17,24 @@
     */
    private static final String HOT_ROD_ENDPOINT_SERVICE = "caching-service-app-hotrod";
 
+   // This should match the value specified for the APPLICATION_USER parameter when creating the caching-service
+   private static final String USERNAME = "test";
```

Comment:
Could you please change the comment style in line 18 as well? It will look consistent.

Reply:
I've also added the APPLICATION_NAME variable to hopefully make things even more explicit